### ISSUE TITLE
关于二级目录的问题

### ISSUE THREAD

**Issue by fxzxmic** [cloudreve/frontend#44]
我曾经在主仓库说明过这个问题，但是开发者回应称在GO语言中不好实现，但是经过我的测试，其实二级目录使用出现问题绝大部分的原因是在前端这里，后端应该是没问题的。
我使用的是Apache，在例如a.com/aaa这个目录下反代Cloudreve，但是网页发起所有请求全部指向了a.com（指向a.com/aaa才可以正常使用），在package.json中添加”homepage“: "/aaa"确实可以使得静态文件正常加载，但是静态文件内部发出的请求还是全部指向了a.com。所以，是否可以为二级目录下的使用提供兼容呢？
其实只需要为js设置一个二级目录的全局参数，在发起js请求的时候将这个参数加入到路径前就可以，或许还需要其它调整，不过还是希望能够兼容一下。

**Comment by topjohncian** [comment]
`homepage` 填写为 `aaa` 呢

**Comment by fxzxmic** [comment]
> `homepage` 填写为 `aaa` 呢

不行。`homepage`只会更改静态资源的目录，不会更改诸如`/api/`之类发起请求的地址。

**Comment by svier** [comment]
确实 后端使用反代没有什么问题 问题基本就在前端上 目前只能修改前端的源码 在每个请求地址前加上/aaa 但是不利于快速配置 如果源码中支持从配置文件兼容二级目录的话方便很多

**Comment by wnpllrzodiac** [comment]
支持nginx proxy 就好了。这样一个主域名可以挂很多子服务，每个服务都要单独开端口不便于维护
